cpp/oneapi/dal/algo/svm/backend/gpu/infer_kernel.cpp

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/algo/svm/backend/gpu/infer_kernel.hpp"
+
+namespace oneapi::dal::svm::backend {
+
+template <typename Float>
+struct infer_kernel_gpu<Float, task::classification, method::by_default> {
```

Comment:
It's not critical, but in specializations/explicit instantiations I'd prefer to use exact `method` tags rather than aliases

Reply:
For the infer, we do not want to create a separate method. So, I decided this trick by default value. Maybe there is a better way?